packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -184,6 +205,16 @@ declare global {
         fragment: T,
         config: MountFragmentConfig<T>
       ): Cypress.Chainable<ClientTestContext>
+
+      /**
+       * mock a mutation resolver when needed to spy on it or modify the result
+       * @param document
+       * @param resolver
+       */
+      setMutationResolver<Result, Variables, T extends TypedDocumentNode<Result, Variables>>(
```

Comment:
I wonder if `mockMutationResolver` or `stubMutationResolver` would be more accurate?

Reply:
`stub` could be more precise, I will fix it.